src/main/interface/msp.c

```diff
@@ -1680,8 +1680,8 @@ static mspResult_e mspProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         sbufReadU8(src); // reserved
         currentPidProfile->vbatPidCompensation = sbufReadU8(src);
         currentPidProfile->setpointRelaxRatio = sbufReadU8(src);
-        currentPidProfile->dtermSetpointWeight = sbufReadU8(src);
```

Comment:
Same deal here. Leave this line as it is, then check if two more bytes are available at the end, and if yes, overwrite `currentPidProfile->dtermSetpointWeight` with the U16 in them.

Reply:
You're unlikely to hit it